pravega/pravega
# Issue 6179: Implement scope list command on pravega-cli

**Change log description**  
Add `scope list` command as part of the commands used in pravega-cli and when called, the existing scopes are displayed alphabetically.

**Purpose of the change**  
Fixes #6179 

**What the code does**  
- Implements the `scope list` command in ScopeCommand.java by traversing through the list of scopes with the scope iterator.
- Sorts the list of scopes in ControllerImpl.java
- Added testListScope method in ScopeCommandsList.java to test out the `scope list` command.

**How to verify it**  
- Run the testListScope() method in ScopeCommandTests.java
- Run the pravega-cli and test manually to see if the scopes do get listed alphabetically.
